README.md

```diff
@@ -1 +1,49 @@
 # OTP Builds
+
+This is a collection of community-maintained [Erlang/OTP](https://github.com/erlang/otp) binary builds.
+
+Supported operating systems:
+
+  * `macos`
+
+Supported architectures:
+
+  * `amd64`
+  * `arm64`
+
+## List of Builds
+
+List of builds can be found here:
+
+  * `macos-amd64` - <https://github.com/erlef/otp_builds/raw/builds/macos-amd64.txt>
+  * `macos-arm64` - <https://github.com/erlef/otp_builds/raw/builds/macos-arm64.txt>
+
+Entries in the list of builds follow this pattern:
+
+    {ref_name} {ref} {datetime} {sha256}
+
+Where `{ref_name}` is the Erlang/OTP git tag or branch name, `{ref}` is the git sha corresponding
+to the `{ref_name}`, `{datetime}` is the time the build was created and `{sha256}` is the build
+SHA-256 checksum.
+
+Example `builds/macos-arm64.txt` entry:
```

Comment:
Should we mention something similar to 

"
At a very minimum, when consuming the images generated by this repository, we strongly suggest you
verify their SHA sum the same way we generate it ("<method>" found next to `.tar.gz`)
"

Reply:
^ that. I would suggest opening PRs @paulo-ferraz-oliveira , we need to strong signaling here in regard to security.

Reply:
Yeah, I'll wait for @wojtekmach's comments. Whatever he doesn't think is relevant for this PR we can turn into issues and implement later.

Reply:
Thanks, I added the following to the README:

```
After downloading the build you should verify its integrity against builds csv mentioned in the previous section, for example:

    $ curl -fLO https://github.com/erlef/otp_builds/releases/download/OTP-27.1.2/OTP-27.1.2-aarch64-apple-darwin.tar.gz
    $ checksum=`curl -fsSL https://github.com/erlef/otp_builds/raw/builds/aarch64-apple-darwin.csv | grep OTP-27.1.2, | cut -d"," -f4`
    $ sha256 OTP-27.1.2-aarch64-apple-darwin.tar.gz --check $checksum
```